Dockerfile

```diff
@@ -1,41 +1,40 @@
-# This docker container can be pulled from umaprotocol/voting on dockerhub.
-# To get the latest image, run:
-#   docker pull umaprotocol/voting
-#
-# Execute the voting system with:
-#   docker run
-#     --env SENDGRID_API_KEY=<key>
-#     --env NOTIFICATION_FROM_ADDRESS=<email address>
-#     --env NOTIFICATION_TO_ADDRESS=<email address>
-#     --env LOCALHOST=<ethereum host>
-#     --env LOCALPORT=<ethereum port>
-#     --env MNEMONIC=<mnemonic>
-#     --env COMMAND="while true; do $(npm bin)/truffle exec ./scripts/Voting.js --network=<your_network>; sleep 60; done"
-#     umaprotocol/voting
+# This docker container can be pulled from umaprotocol/protocol on dockerhub.
```

Comment:
notice the repo name change.

I updated this comment to reflect a more up to date example.